recipes/foxglove-schemas-protobuf/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy, apply_conandata_patches, export_conandata_patches
+from conan.tools.microsoft import check_min_vs
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class FoxgloveSchemasProtobufConan(ConanFile):
+    name = "foxglove-schemas-protobuf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/foxglove/schemas"
+    description = "Protobuf schemas for Foxglove"
+    license = "MIT"
+    topics = ("foxglove", "protobuf", "schemas")
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps"
+    package_type = "library"
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16.9",
```

Comment:
I think this is not a valid version in `settings.yml`, unless I'm missing something

Reply:
I think this came over when I copied a different recipe. I'll take it out since I don't even know if it applies.